sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/ValueEnricher.scala

```diff
@@ -86,7 +86,7 @@ final class ValueEnricher(
 
   private[this] def handleLookup[X](lookup: => Either[LookupError, X]) = lookup match {
     case Right(value) => ResultDone(value)
-    case Left(LookupError.MissingPackage(pkgId, context)) =>
+    case Left(LookupError.MissingPackage(PackageRef.Id(pkgId), context)) =>
```

Comment:
One case where we have different handling for missing package by id vs name, and can distinguish easily enough.